### PR TITLE
Adding the to_JSON function

### PR DESCRIPTION
**Pull request by oualib**
UT will be done in the next days.

### CODE REVIEW

File: verticapy/learn/ensemble.py [hunk 1]
```diff
@@ -60,6 +62,167 @@
...
 
+# ---#
+class XGBoost_to_json:
+    # Class to export Vertica XGBoost to the Python XGBoost JSON format.
+    def to_json(self, path: str = ""):
+        """
+        ---------------------------------------------------------------------------
+        Creates the JSON file used to import it in the Python XGBoost API.
+        
+        \u26A0 Warning : In case of multiclass classifiers, the probabilities
```
Comment (afard) [inline verticapy/learn/ensemble.py:73]: What is `\u26A0`?

File: verticapy/learn/ensemble.py [hunk 2]
```diff
@@ -60,6 +62,167 @@
...
+            nothing is returned.
+        """
+        def xgboost_to_json(model):
+            def xgboost_dummy_tree_dict(model, c: str = None):
+                condition = ["{} IS NOT NULL".format(elem) for elem in model.X] + ["{} IS NOT NULL".format(model.y)]
+                model.cursor.execute("SELECT COUNT(*) FROM {} WHERE {} AND {} = '{}'".format(model.input_relation, " AND ".join(condition), model.y, c))
+                avg = model.cursor.fetchone()[0]
+                model.cursor.execute("SELECT COUNT(*) FROM {} WHERE {}".format(model.input_relation, " AND ".join(condition),))
+                avg /= model.cursor.fetchone()[0]
+                split_conditions = [np.log(avg / (1 - avg))]
```
Comment (afard) [inline verticapy/learn/ensemble.py:98]: Are you calculating a priori probability?
A line of comments would be nice to explain the purpose of xgboost_to_json.

**Review by afard — APPROVED** [review]
I approve it for now. Please add in-code comments in your next PR when you add unit tests.

**Review by ansleis — APPROVED** [review]

File: verticapy/learn/ensemble.py (hunk 2 above)
Comment (oualib) [reply verticapy/learn/ensemble.py:99]: yes I am. OK it will be added.

**Pull request merged**